POUF1.md

```diff
@@ -0,0 +1,442 @@
+
+Additionally, this POUF supports custom delegated targets roles and terminating delegations, features described as optional in the [Standard](https://uptane.github.io/uptane-standard/uptane-standard.html#targets_role_delegations). As explained in the Standard, terminating delegations are not used by default. To use custom roles, the Targets role on the Image repository sets up a delegation. Delegations used in this POUF must support multi-role delegations, as explained in [TAP 3](https://github.com/theupdateframework/taps/blob/d0818e580c322815a473520f2e8cc5f5eb8df499/tap3.md). In addition, delegation filenames are listed using an enumerated list that supports [wildcards](https://uptane.github.io/uptane-standard/uptane-standard.html#delegations_meta). Delegations are supported in the reference implementation to demonstrate how they can be used to establish a chain of trust for specific images.
+
+Public key identifiers use the hash of the key value, encryption algorithm, and verification scheme for a condensed identifier. The format of these identifiers if further described in {Formats}. This POUF supports rsa and ed25519 hashes for public keys. The hashing algorithm must be specified in the signature method.
+
+This POUF supports asymmetric keys and uses the [same keys for encrypting and signing](https://uptane.github.io/uptane-standard/uptane-standard.html#build-time-prerequisite-requirements-for-ecus). Using the same keys allows for a simplified reference implementation and less key management. In particular, each ECU has a private key and registers the associated public key with the director repository. The ECU will use the private key to sign messages and the director repository may use the public key to encrypt images per ECU. In addition, each metadata role has either online or offline private keys that are used on the director and image repositories. The public keys associated with these roles are used by the ECUs during validation.
+
+The Image repository and Director repository have public interfaces for communication with ECUs. To make the public interface, the Image and Directory repositories each store metadata in a filesystem that is hosted to form the public interface. These interfaces do not require authentication to read.
+
+The Director repository identifies a vehicle at the start of the update process using a stored vehicle version manifest. The Director checks the vehicle version manifest as described in the standard, but does not make any additional checks. This POUF does not do dependency resolution as the reference implementation is not used in practice and so does not have conflicts.
```

Comment:
A few odd things here.  What if an ECU was replaced?  What do you mean about dependency resolution here?  This is for packages on a single ECU.  What does your statement have to do with conflicts.

Reply:
I clarified this a bit and removed the reference to dependency resolution. Dependency resolution seems to be out of scope for this document as it does not affect any of the communication and the standard already requires that some form of dependency resolution is used.